snippets/html.json

```diff
@@ -6,7 +6,8 @@
 			"<!--[if lt IE 7]>      <html class=\"no-js lt-ie9 lt-ie8 lt-ie7\"> <![endif]-->",
```

Comment:
There is already an html opening tag here!

Reply:
Yep. It was more of out of the convenience to have the simple <html> tag (similar to another vscode extension that I was using instead). More visually clear for a repetitive task, but obviously not a big deal.